client/src/motor.ts

```diff
@@ -0,0 +1,9 @@
+export enum Motor {
```

Comment:
It might be clearer to call this `ArmMotor` since the wheels are also technically motors. This would clear up some possible confusion on line 107 of the `update()` method below.